test/test_utils.py

```diff
@@ -84,11 +84,11 @@ def test__get_archive_filelist_nonexistent_file(self):
         # tarfile.is_tarfile() throws an IOError in Python2.7 and
         # FileNotFoundError  in Python3.6
         try:
-            FileNotFoundError
+            expected_err = eval('FileNotFoundError')
```

Comment:
If FileNotFoundError is not defined, as it is in v2, leaving it as a bare identifier causes an error in a static analysis tool.